spec/features/publish/courses/new_tda_course_spec.rb

```diff
@@ -250,20 +273,44 @@ def then_i_am_on_the_choose_schools_page
     expect(page).to have_current_path(new_publish_provider_recruitment_cycle_courses_schools_path(provider_code: provider.provider_code, recruitment_cycle_year: 2025), ignore_query: true)
   end
 
+  def and_the_back_link_points_to_outcome_page
+    and_i_click_back
+    expect(page).to have_current_path(new_publish_provider_recruitment_cycle_courses_outcome_path(provider_code: provider.provider_code, recruitment_cycle_year: 2025), ignore_query: true)
+    and_i_click_continue
+  end
+
   def when_i_choose_the_school
     check provider.sites.first.location_name
     and_i_click_continue
   end
 
+  def then_the_back_link_points_to_the_school_page
+    and_i_click_back
+    expect(page).to have_current_path(new_publish_provider_recruitment_cycle_courses_schools_path(provider_code: provider.provider_code, recruitment_cycle_year: 2025), ignore_query: true)
```

Comment:
I think we should add a clean up card, for removing the feature flag and when we are in the next cycle we should make these recruitment cycles dynamic.

Happy to create one.

Reply:
Agreed. Yes, can you create one please? ❤️

Reply:
https://trello.com/c/v5PXt7m3/1736-remove-tda-feature-flag

Feel free to add/changed as necessary.